packages/itwinui-react/src/core/Tree/TreeNode.tsx

```diff
@@ -63,10 +87,18 @@ type TreeNodeProps = {
    * Recommended to use `Checkbox` component.
    */
   checkbox?: React.ReactNode;
+  /**
+   * Props for TreeNode checkbox.
+   */
+  checkboxProps?: React.ComponentProps<'div'>;
   /**
    * Custom expander element. If `hasSubNodes` is false, it won't be shown.
    */
   expander?: React.ReactNode;
+  /**
+   * Props for content
```

Comment:
This jsdoc should also be updated to more clearly describe which "content".

Reply:
Updated in this [commit](https://github.com/iTwin/iTwinUI/pull/2022/commits/437d7379210c8f1022bc0ba9e03d847f8031a3df).

Reply:
slight grammatical nit: "effects" (noun) → "affects" (verb)

Reply:
Fixed in this [commit](https://github.com/iTwin/iTwinUI/pull/2022/commits/b684869778dbe696d7ec70e858a49e67395b4353).